pkg/reconciler/revision/resources/queue.go

```diff
@@ -323,6 +328,9 @@ func makeQueueContainer(rev *v1alpha1.Revision, loggingConfig *logging.Config, t
 		}, {
 			Name:  "INTERNAL_VOLUME_PATH",
 			Value: internalVolumePath,
+		}, {
+			Name:  "DOWNWARD_API_LABELS_PATH",
+			Value: fmt.Sprintf("%s/%s", podInfoVolumePath, metadataLabelsPath),
```

Comment:
Should we only set this if the feature is enabled to allow the queue-proxy to short-circuit the filesystem checks in absence?

Alternatively we can always set it and check in the QP on startup if the respective file is there and set a flag to `false` if it's not.

Reply:
went with your second suggestion.